src/map/utils/zoneutils.cpp

```diff
@@ -430,9 +430,9 @@ namespace zoneutils
                     {
                         while (sql->NextRow() == SQL_SUCCESS)
                         {
-                            ZONE_TYPE zoneType = PZone->GetType();
+                            ZONE_TYPE zoneType = PZone->GetTypeMask();
 
-                            if (zoneType != ZONE_TYPE::DUNGEON_INSTANCED)
+                            if (!(zoneType & ZONE_TYPE::INSTANCED))
```

Comment:
In the future, I think we might want to avoid bit compares in the condition, and perhaps abstract it out to a function.  Out of scope though

Reply:
eventually when I have more time I think we want whats incoming for status effect flag functions: `GetZoneType(Enumhere)` to check single bits of the mask. Just not tonight unless someone else wants to dive in. :)